types/probe-image-size/tsconfig.json

```diff
@@ -14,6 +14,10 @@
     },
     "files": [
         "index.d.ts",
-        "probe-image-size-tests.ts"
+        "probe-image-size-tests.ts",
+        "lib/parse_sync/bmp.js/index.d.ts",
+        "lib/parse_sync/ico.js/index.d.ts",
+        "lib/parse_sync/png.js/index.d.ts",
```

Comment:
Why aren't these `lib/parse_sync/bmp.d.ts`, `ico.d.ts`, etc?

Reply:
Thank you for the suggestion, I've improved the file structure